.github/workflows/test.yml

```diff
@@ -58,3 +58,11 @@ jobs:
     - name: run regression tests
       run: |
        make test
+    - name: code coverage
+      run: |
+       make coverage
+    - name: upload coverage data to coveralls.io
+      run: coveralls
+      env:
+        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_TOKEN }}
+        COVERALLS_PARALLEL: true
```

Comment:
We should report coverage only for one matrix entry. We can use [matrix.include](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) syntax for this ([my example](https://github.com/tarantool/smtp/blob/322254c83169905bc43cb8a25972fa19a03f72ea/.github/workflows/testing.yml#L44-L57)).

Reply:
done